.github/workflows/linter-workflow.yml

```diff
@@ -0,0 +1,21 @@
+name: End-to-End Tests
+on: pull_request
+
+jobs:
+  test:
+    name: Run TestCafe Tests
```

Comment:
what is the rationale in having test cafe testing under the linter workflow?

Reply:
Sorry, you can ignore this PR for now. I am just trying out how to get the linter and tests to run on pull requests and hadn't updated the wording. Going to convert to a draft temporarily as I explore some more documentation.